microsoft-edge/webview2/gettingstarted/winUI.md

```diff
@@ -0,0 +1,293 @@
+          xmlns:local="using:WinUI_Sample"
+          xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+          xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+          mc:Ignorable="d"
+          xmlns:controls="using:Microsoft.UI.Xaml.Controls"
+          >
+        
+          <StackPanel Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Center">
+            <Button x:Name="myButton" Click="myButton_Click">Click Me</Button>
+          </StackPanel>
+
+    </Window>
+    ```  
+    
+1.  Add the WebView2 control by replacing the `<StackPanel>` tags, with the following code snippet.  The `Source` property sets the initial URI displayed in the WebView2 control.  
+    
+    ```xml  
+    <Grid>
+
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="Auto" />
+        </Grid.ColumnDefinitions>
+
+        <controls:WebView2 x:Name="MyWebView"  controls:Grid.Row="1" controls:Grid.ColumnSpan="2" 
+        Source="https://www.microsoft.com" HorizontalAlignment="Stretch" VerticalAlignment="Stretch" />
+
+    </Grid>
+    ```  
+3.  Additionally, open `MainWindow.xaml.cs` and comment out the following line:
+    ```xml
+        //myButton.Content = "Clicked";     
+    ```   
+1.  Press `F5` to build and run your project.  Confirm that your WebView2 control displays [https://www.microsoft.com](https://www.microsoft.com).  
+    
+    :::image type="complex" source="./media/winui-gettingstarted-part3.png" alt-text="Microsoft.com":::
+       Microsoft.com
+    :::image-end:::  
+    
+## Step 4 - Navigation Control
+
+Add the ability to allow users to change the URL that the WebView2 control displays by adding an address bar to the app.
+
+1.  In **MainWindow.xaml**, add an address bar by copying and pasting the following code snippet inside the Grid that contains the WebView.  
+    
+    ```xml
+        <TextBox Name="addressBar" Grid.Column="0"/>
+        <Button x:Name="myButton" Grid.Column="1" Click="myButton_Click">Go</Button>
+    ```  
+    
+    Confirm that the `Grid` section of `MainWindow.xaml` looks like the following code snippet.  
+    
+    ```xml
+    <Grid>
+
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="Auto" />
+        </Grid.ColumnDefinitions>
+    
+        <TextBox Name="addressBar" Grid.Column="0"/>
+        <Button x:Name="myButton" Grid.Column="1" Click="myButton_Click">Go</Button>
+
+        <controls:WebView2 x:Name="MyWebView"  controls:Grid.Row="1" controls:Grid.ColumnSpan="2" 
```

Comment:
This is undoubtedly my fault from when I helped build this UI, but the TextBox and Button don't contain the "controls:" namespace prefix on their Grid.Column properties, but the prefix is on the properties here.  The namespace isn't necessary, though it currently helps IntelliSense. I recommend either adding the namespace for those other controls, or removing them on WebView2, to be consistent one way or the other.  I don't have a strong preference, but I suggest removing the namespace on these properties to keep things simpler.